lldb/test/API/functionalities/target-new-solib-notifications/TestModuleLoadedNotifys.py

```diff
@@ -115,9 +151,10 @@ def test_launch_notifications(self):
         # binaries in batches.  Check that we got back more than 1 solib per event.
         # In practice on Darwin today, we get back two events for a do-nothing c
         # program: a.out and dyld, and then all the rest of the system libraries.
-        # On Linux we get events for ld.so, [vdso], the binary and then all libraries.
-
-        avg_solibs_added_per_event = round(
-            float(total_solibs_added) / float(total_modules_added_events)
+        # On Linux we get events for ld.so, [vdso], the binary and then all libraries,
+        # but the different configurations could load a different number of .so modules
+        # per event.
+        self.assertGreaterEqual(
+            len(set(max_solib_chunk_per_event).intersection(expected_solibs)),
+            len(expected_solibs),
```

Comment:
I think this could be `assertLess(set(expected_solibs), set(max_solib_chunk_per_event))` (with the advantage being that you'd bet a better error message than `3 is not less than 4` when it fails.

Reply:
Done.